app/views/dashboards/_dashboard_article.html.erb

```diff
@@ -41,32 +41,32 @@
   </div>
   <div class="dashboard-actions">
     <% if !article.published %>
-      <a href="<%= article.current_state_path %>" class="pill cta yellow">DRAFT</a>
+      <a href="<%= article.current_state_path %>" class="crayons-btn">Draft</a>
     <% end %>
-    <a href="<%= article.path %>/edit" class="pill cta green">EDIT</a>
+    <a href="<%= article.path %>/edit" class="crayons-btn">Edit</a>
     <% if manage_view %>
-      <a href="<%= article.path %>/delete_confirm" data-no-instant class="cta pill black">DELETE</a>
+      <a href="<%= article.path %>/delete_confirm" data-no-instant class="crayons-btn crayons-btn--danger">Delete</a>
       <% if article.profile_pins.any? %>
         <%= form_for(article.profile_pins.first) do |f| %>
-          <button class="pill cta">UNPIN</button>
+          <button class="crayons-btn crayons-btn--secondary">UNPIN</button>
         <% end %>
       <% else %>
         <%= form_for(ProfilePin.new) do |f| %>
           <%= f.hidden_field :pinnable_id, value: article.id %>
-          <button class="pill cta">PIN TO PROFILE</button>
+          <button class="crayons-btn">Pin to profile</button>
         <% end %>
       <% end %>
     <% elsif article.published %>
-      <a href="<%= article.path %>/manage" class="cta pill black">MANAGE</a>
+      <a href="<%= article.path %>/manage" class="crayons-btn">Manage</a>
```

Comment:
I think that when we circle back on this view and apply Crayons to the entire view, some of the button classes I'm using here might be different.